src/nvim/ex_cmds.c

```diff
@@ -381,6 +382,13 @@ typedef struct {
   } st_u;
 } sorti_T;
 
+static int string_compare(const void *s1, const void *s2) FUNC_ATTR_NONNULL_ALL
+{
+  if (sort_lc) {
+    return strcoll((char *)s1, (char *)s2);
+  }
+  return sort_ic ? STRICMP(s1, s2) : STRCMP(s1, s2);
```

Comment:
What's the point of check `sort_lc` twice when `strcoll` doesn't modify it?
This came from the patch but it looks wrong to me.

Reply:
This checks `sort_ic` (for case-insensitivity), not `sort_lc`.